setup/install/providers/aws/aws-ec2.md

```diff
@@ -85,55 +93,234 @@ aws cloudformation deploy --stack-name spinnaker-managed-infrastructure-setup --
 --parameter-overrides AuthArn=FROM_ABOVE ManagingAccountId=FROM_ABOVE --capabilities CAPABILITY_NAMED_IAM --region us-west-2
 ```
 
-## Option-3 : Use AWS Console UI (Manual Steps) 
+## Option-3 : Configure with AWS IAM Console
+The Spinnaker **Managing** role AWS account Assumes the Spinnaker **Managed** role AWS account in a target AWS account via AWS IAM resources (policies, roles, users, trust relationship, etc). This allows Spinnaker to take control of the AWS cloud resources.
+
+For the Example below the AWS Account **spinnakerManaging** assumes the **spinnakerManaged** role in the AWS accounts **develop** and **staging**. The account **spinnakerManaging** is where Spinnaker lives
```

Comment:
```suggestion
For the Example below the AWS Account **spinnakerManaging** assumes the **spinnakerManaged** role in the AWS accounts **develop** and **staging**. The account **spinnakerManaging** is where Spinnaker lives.
```

Reply:
"...where Spinnaker _lives_."

Is there a more descriptive term than "lives"? Is it the account Spinnaker _uses_ to access AWS resources?

Reply:
@dorbin  Is the Account where Spinnaker is deployed and running.